jans-config-api/server/src/main/java/io/jans/configapi/service/auth/UsrService.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Janssen Project software is available under the Apache License (2004). See http://www.apache.org/licenses/ for full text.
+ *
+ * Copyright (c) 2020, Janssen Project
+ */
+
+package io.jans.configapi.service.auth;
+
+import io.jans.as.common.model.common.User;
+import io.jans.as.common.service.common.UserService;
+import io.jans.as.common.util.AttributeConstants;
+import io.jans.as.model.config.StaticConfiguration;
+import io.jans.as.model.configuration.AppConfiguration;
+import io.jans.configapi.rest.model.SearchRequest;
+import io.jans.orm.model.PagedResult;
+import io.jans.orm.model.SortOrder;
+import io.jans.orm.search.filter.Filter;
+import static io.jans.as.model.util.Util.escapeLog;
+
+import java.util.List;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+
+@ApplicationScoped
+@Named("userSrv")
+public class UsrService extends UserService {
```

Comment:
`public class UserService extends io.jans.as.common.service.common.UserService` will work too.

Reply:
it was `public class UserService extends io.jans.as.common.service.common.UserService` but when analyzed it locally it was giving sonar issue to change the name and hence i changed it to ` UsrService`.
Will check in as suggested and lets see

Reply:
I see, let it be then.

Reply:
Strangely it is giving error locally but not reported by Github sonar check
![image](https://user-images.githubusercontent.com/43700552/161938605-069fd351-9ad1-4567-8b01-71c44351ddfe.png)